apps/fluid-dev/src/pages/switch/switch-page.component.html

```diff
@@ -1,6 +1,7 @@
 <fluid-switch
   [disabled]="disabled"
   [checked]="checked"
+  [value]="'switch'"
```

Comment:
No need to bind that property, if you don't bind it to a variable.
```suggestion
  value="switch"
```